tools/profiling/templates/doubles.tpl

```diff
@@ -35,7 +35,7 @@
             {sql_queries data=$nb}
           </td>
           <td class="pre">
-            <pre>{$q}</pre>
+            <pre>{$q nofilter}</pre>
```

Comment:
WDYT @PierreRambaud ?

Reply:
@eternoendless  @PierreRambaud 
`$q` always contains HTML, for highlighting the values replaces by `XX`. If you feel that `nofilter` could be dangerous, then we need another way to highlight the blue text.

Reply:
Thanks, PR updated. The nofilter is not really needed here, the code added is useless and I can add an XSS because of this.

Reply:
XSS inside the query coming from the software in the profiler displayed when the developer explicitly tells the application to do it? 🤔